Geometry/HGCalCommonData/plugins/DDHGCalEEAlgo.cc

```diff
@@ -281,8 +281,10 @@ void DDHGCalEEAlgo::positionSensitive(const DDLogicalPart& glog,
       int iv = std::abs(v);
       int nr = 2 * v;
       int nc = -2 * u + v;
-      double xpos = xyoff.first + nc * r;
-      double ypos = xyoff.second + nr * dy;
+      double xps0 = nc * r;
```

Comment:
it seems like these new intermediate variables are unnecessary now?

Reply:
No they are not. I was waiting to see Federico's OK